.github/workflows/ci_benchmark.yml

```diff
@@ -0,0 +1,99 @@
+### Copied from https://github.com/scikit-image/scikit-image/blob/main/.github/workflows/benchmarks.yml
+
+name: Benchmark
+
+on:
+  pull_request:
+    types: [labeled]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  benchmark:
+    if: contains(github.event.label.name, 'benchmark') || github.event_name == 'workflow_dispatch'
```

Comment:
We have a lot of old forks with Actions unintentionally enabled. Also someone might come add more triggers in the future, so I would feel better if we are stricter with the `if`, maybe similar to this:

https://github.com/astropy/astropy/blob/3d5aae9d0b435c459d4e10c4958c73568eda6552/.github/workflows/ci_cron_weekly.yml#L33